vendor/nsjail/dockerfiles/Dockerfile.ubuntu-20.04

```diff
@@ -0,0 +1,48 @@
+ARG UBUNTU_VERSION="20.04"
```

Comment:
this doesn't build for me, seems like there is a bug in ubuntu kernel headers

Reply:
I didnt mean to include that in this pr 😅